adal/build.gradle

```diff
@@ -229,9 +229,37 @@ publishing {
         }
     }
 
+    // Repositories to which Gradle can publish artifacts
+    if (System.getenv("VSTS_ENV_ACCESS_TOKEN") != null || project.hasProperty("vstsMavenAccessToken")) {
+        repositories {
+            maven {
+                name "vsts-maven-adal-android"
+                url "https://identitydivision.pkgs.visualstudio.com/_packaging/AndroidADAL/maven/v1"
+                credentials {
+                    username "VSTS"
+                    password System.getenv("VSTS_ENV_ACCESS_TOKEN") != null ? System.getenv("VSTS_ENV_ACCESS_TOKEN") : vstsMavenAccessToken
+                }
+            }
+        }
+    } else {
+        repositories {
+            maven {
+                url "\\\\adalbuildvm\\AndroidRelease\\repository"
+            }
+        }
+    }
+}
+
+// Repositories from which Gradle can fetch dependencies
+if (System.getenv("VSTS_ENV_ACCESS_TOKEN") != null || project.hasProperty("vstsMavenAccessToken")) {
```

Comment:
Does adal need to fetch any dependencies from the vsts shared repo?

Reply:
No, we don't.

Reply:
then can you remove this entire section?